nautobot_chatops_ipfabric/ipfabric.py

```diff
@@ -324,3 +324,46 @@ def get_wireless_ssids(self, snapshot_id="$last", limit=DEFAULT_PAGE_LIMIT):
         }
 
         return self.get_response("/api/v1/tables/wireless/radio", payload)
+
+    def get_host(self, search_key, search_value, snapshot_id="$last", limit=DEFAULT_PAGE_LIMIT):
+        """Return inventory host information."""
+        logger.debug("Received host inventory request")
```

Comment:
maybe we could also log the arguments? (search_key and search_value)

Reply:
will update this